### PR TITLE
CDC engine

### PR DESCRIPTION
**Pull request by angrykoala**
# Description

This PR introduces the CDC engine, allowing consumption of CDC events through GraphQL subscriptions.

This come with a couple of caveats:
* Only Databases with CDC enabled (5.13) will support it
* Relationships events are disabled

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Made some changes to the Neo4j GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at https://neo4j-graphql-toolbox-4194.surge.sh

Use for instance a local dbms hosted on [Neo4j Desktop](https://neo4j.com/download/) or a cloud-hosted [Aura Free](https://neo4j.com/cloud/aura-free/) dbms.

File: packages/graphql/src/classes/subscription/cdc/cdc-event-parser.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        //     schemaModel: this.schemaModel,
+        // });
+        // if (!startTypenames || !startTypenames[0]) return undefined; // What happens with multiple typenames?
+
+        // // const startTypename = startTypenames[0];
+        // const startEntity = this.schemaModel.getEntitiesByLabels(cdcEvent.start.labels);
+
+        // // TODO: Get relationship
+        // console.log(cdcEvent);
+        // return undefined;
```
Comment (a-alle) [inline packages/graphql/src/classes/subscription/cdc/cdc-event-parser.ts:107]: is this comment needed?

**Review by a-alle — APPROVED** [review]

File: packages/graphql/src/classes/subscription/cdc/cdc-event-parser.ts (hunk 1 above)
Comment (angrykoala) [reply packages/graphql/src/classes/subscription/cdc/cdc-event-parser.ts:107]: mmm, not really, but this is the logic to capture relationship events, of which only the start/end node properties is missing

Comment (a-alle) [reply packages/graphql/src/classes/subscription/cdc/cdc-event-parser.ts:107]: I see

**Pull request merged**